SolutionItems/Properties/AssemblyInfoGlobal.cs

```diff
@@ -4,10 +4,10 @@
 
 
 [assembly: AssemblyCopyright("Copyright � 2014")]
-[assembly: AssemblyVersion("0.1.3.1")]
+[assembly: AssemblyVersion("0.1.3.11")]
```

Comment:
should be 0.1.4.0 since it adds additional functionality, I'll be creating a 1.0 release soon.